test/accelerators/NNPA/backend/CMakeLists.txt

```diff
@@ -152,13 +152,13 @@ set(NNPA_TEST_LIST
     # Disable backend tests for ConvTranspose, because they do not work on CPU.
     # test_convtranspose_1d_cpu,zdnn_conv1d
     # test_convtranspose_3d_cpu
-    test_convtranspose_autopad_same_cpu,zdnn_conv2d,NO_DYNAMIC_SHAPE_TEST # not work on CPU
-    test_convtranspose_cpu,zdnn_conv2d,NO_DYNAMIC_SHAPE_TEST # not work on CPU
+    test_convtranspose_autopad_same_cpu,zdnn_conv2d,NO_DYNAMIC_SHAPE_TEST # Spatial dims must be static even on CPU.
```

Comment:
Can we use `DYNAMIC_SHAPE` to specify which dims are dynamic on CPU? so that spatial dims are always static.

Reply:
I set dynamic shape in spacial dims for input on CPU, but it didn't work. DYNAMIC_SHAPE option modify only shape of input to dynamic, but I found we need to modify also the shape of output. 
It seems the shape mismatch between the output of rewriting and model output. There may be a bug in rewriting.

Reply:
@tungld  In dynamic backend test, input becomes dynamic, but output does not. Current rewriting fails to replace output tensor  because output tensor dimension by rewriting and existing dimension are different. Also shapeinference fails to replace output tensor. Should we be able to replace in this case?

```
 func.func @main_graph(%arg0: tensor<?x?x3x3xf32> {onnx.name = "X"}, %arg1: tensor<?x?x3x3xf32> {onnx.name = "W"}) -> (tensor<1x2x6x6xf32> {onnx.name = "Y"}) {
    %0 = "onnx.NoValue"() {value} : () -> none
    %1 = "onnx.ConvTranspose"(%arg0, %arg1, %0) {auto_pad = "SAME_UPPER", group = 1 : si64, strides = [2, 2]} : (tensor<?x?x3x3xf32>, tensor<?x?x3x3xf32>, none) -> tensor<1x2x6x6xf32>
    onnx.Return %1 : tensor<1x2x6x6xf32>
```

Reply:
It's the problem of dynamic tests. When changing the input tensor to dynamic, we should change the output tensor to a unranked tensor, and let the shape inference does its job so that we can check if the shape inference works for dynamic tensor also. If an output tensor has ranked, the compiler will consider its the correct shape and the shape inference may fail due to the mismatched shape.

Reply:
Removed `NO_DYNAMIC_SHAPE_TEST` because PR #2834 was merged. Thanks!